packages/foreigntypes/posixtypes.pony

```diff
@@ -0,0 +1,95 @@
+// This file defines the standard Posix type to make FFI calls easier
+// to manage and more consistent.
+
+type BlkCnt is I64
+
+// not consistent across OSX (always 32 bit int)/Linux (Long)
+type BlkSize is ILong
+type OSXBlkSize is I32
```

Comment:
This was my crappy suggestion. The problem is that ponyc tries very hard indeed to type check on all platforms, even though it compiles for just one at a time. @andymcn put a ton of work into that. Adding platform specific types would be... tricky. I'm all for it in theory, but it seems to constitute a combinatorial attack on the type checker :)